modules/routing/eval/src/query_generator.cc

```diff
@@ -260,8 +260,11 @@ int main(int argc, char const** argv) {
   parser.print_unrecognized(std::cout);
   parser.print_used(std::cout);
 
-  std::ofstream out_fwd(generator_opt.target_file_fwd_);
-  std::ofstream out_bwd(generator_opt.target_file_bwd_);
+  utl::verify(
```

Comment:
Wäre es nicht einfacher, die Filenames aus den Targets abzuleiten?
substr(1).replace("/" durch "_")

Reply:
Hatte ich auch überlegt, aber mit der Einzeilerlösung wird es dann unschönes wie:

responses.txt -> responses.txt_routing
responses.txt -> responses.txt_raptor_cpu

Also müsste man die Targets vor die filename extension packen (falls eine existiert) oder man hängt sie immer hinten dran und hört auf filename extensions zu benutzen.

Reply:
Query File: ${escaped target}_queries.txt
Response File: ${escaped target}_responses.txt

Beispiel: csa_cpu_sse_queries.txt -> csa_cpu_sse_responses.txt

D.h. im batch Modus gibt es auch keine Möglichkeit mehr, den Namen der Ausgabedatei festzulegen?

Reply:
Ja, escapes sind eine Lösung. Fand ich irgendwie overkill, aber ist wohl doch die beste Lösung.

Deine Frage verstehe ich nicht. Wieso sollte das nicht möglich sein?

Reply:
Möglich ist es auf jeden Fall. Meine Frage war eher: will man das? Weil wenn jeder sowieso ${escaped target}_responses.txt eingibt, kann man das auch programmatisch festlegen. Aber vll. ist es besser, das dort so zu lassen aus dem Grund, dass man die Datei in einem anderen Verzeichnis erzeugen möchte.